.travis.yml

```diff
@@ -46,6 +46,22 @@ before_install:
   - mongod --version
   - mongo-orchestration --version
   - export MO_PATH=`python -c 'import mongo_orchestration; from os import path; print(path.dirname(mongo_orchestration.__file__));'`
+  - |
+    # tpecl is a helper to compile and cache php extensions
+      tpecl () {
+          local ext_name=ext-mongodb
+          local ext_so=mongodb.so
+          local ext_dir=$(php -r "echo ini_get('extension_dir');")
+          local ext_cache=~/php-ext/$(basename $ext_dir)/$ext_name
+          if [[ -e $ext_cache/$ext_so ]]; then
+              echo extension = $ext_cache/$ext_so >> $INI
+          else
+              mkdir -p $ext_cache
+              echo yes | pecl install -f $ext_name &&
+              cp $ext_dir/$ext_so $ext_cache
+          fi
+      }
+      export -f tpecl
```

Comment:
`before_script` below is still using `pecl install`, so that will need to be changed to `tpecl` (see: https://github.com/symfony/symfony/blob/32cc2e06a326b1dd888c02e1cc48988a14cb9a65/.travis.yml#L137). I don't think we need to bother with `tfold` right now to [collapse Travis CI log output](https://blog.travis-ci.com/2013-05-22-improving-build-visibility-log-folds/); that's something we can always follow up with in the future.

Calling `tpecl` with the extension name (usual argument to `pecl install`) and `.so` filename should suffice:

```
tpecl mongodb-${DRIVER_VERSION} mongodb.so
```